packages/webpack-cli/lib/groups/ConfigGroup.js

```diff
@@ -143,18 +143,34 @@ class ConfigGroup extends GroupHelper {
 
     async resolveConfigFiles() {
         const { config, mode } = this.args;
-        if (config) {
-            const configPath = resolve(process.cwd(), config);
-            const configFiles = getConfigInfoFromFileName(configPath);
-            if (!configFiles.length) {
-                throw new ConfigError(`The specified config file doesn't exist in ${configPath}`);
+        if (config.length > 0) {
+            const resolvedOptions = [];
+            const finalizedConfigs = config.map(async (webpackConfig) => {
+                const configPath = resolve(process.cwd(), webpackConfig);
+                const configFiles = getConfigInfoFromFileName(configPath);
+                if (!configFiles.length) {
+                    throw new ConfigError(`The specified config file doesn't exist in ${configPath}`);
```

Comment:
I think we can improve the message.
```js
if (!configFiles.length) {
    throw new ConfigError(`The specified config file ${basename(configPath)} doesn't exist in ${process.cwd()}`);
                }
```

WDYT?

Reply:
we shouldn't do this, user can pass config path as well, will look redundant